test/RouterUniV2V3.js

```diff
@@ -0,0 +1,170 @@
+const { ethers } = require('hardhat');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+const { ether, constants } = require('@1inch/solidity-utils');
+const { ProtocolKey } = require('./helpers/utils');
+const { initRouterContracts } = require('./helpers/fixtures');
+const { createGasUsedTable } = require('./helpers/table');
+const { MixedRouteTrade, MixedRouteSDK, Protocol, Trade, MixedRoute  } = require('@uniswap/router-sdk');
+const { SwapRouter, UniswapTrade } = require('@uniswap/universal-router-sdk');
+const { Pool } = require('@uniswap/v3-sdk');
+const { Pair } = require('@uniswap/v2-sdk');
+const { CurrencyAmount, Token, TradeType, Ether, Percent } = require('@uniswap/sdk-core');
+
+Error.stackTraceLimit = Infinity
+
+describe('Router [UniV2 => UniV3]', async function () {
+    const gasUsedTable = createGasUsedTable("UniswapV2 => UniswapV3 pools", "path");
+
+    const poolsV2 = {
```

Comment:
What do you think about moving these (`poolV2` and `poolV3`) constants to a separate file in the `helpers` directory and reuse it in appropriate tests?

Reply:
sure, they are used in other tests so it'd make sense to not repeat them.

Reply:
added in the latest push, let me know if the naming scheme is okay